packages/yew-macro/src/html_tree/html_dashed_name.rs

```diff
@@ -16,6 +16,17 @@ pub struct HtmlDashedName {
 }
 
 impl HtmlDashedName {
+    /// Checks if this name is equal to the provided item (which can be anything implementing
+    /// `Into<String>`).
+    pub fn eq<S>(&self, other: S) -> bool
```

Comment:
Calling this `eq` is a bit confusing because `HtmlDashedName` already implements `PartialEq` which doesn't behave the same way.
Maybe we could call it `eq_ignore_ascii_case` mirroring [str](https://doc.rust-lang.org/std/primitive.str.html#method.eq_ignore_ascii_case)?

Reply:
Yup that sounds clearer :)